elasticsearch_dsl/search.py

```diff
@@ -726,6 +726,53 @@ def scan(self):
         for hit in scan(es, query=self.to_dict(), index=self._index, **self._params):
             yield self._get_result(hit)
 
+    def page(self):
+        """
+        Turn the search into a paged search utilizing Point in Time (PIT) and search_after.
+        Returns a generator that will iterate over all the documents matching the query.
+        """
+        search = self._clone()
+
+        # A sort is required to page search results. We use the optimized default if sort is None.
+        # https://www.elastic.co/guide/en/elasticsearch/reference/current/paginate-search-results.html
+        if not search._sort:
+            search._sort = ["_shard_doc"]
+
+        keep_alive = search._params.pop("keep_alive", "30s")
+        pit = search._using.open_point_in_time(
+            index=search._index, keep_alive=keep_alive
+        )
+        pit_id = pit["id"]
+
+        # The index is passed with Point in Time (PIT).
+        search._index = None
+        search._extra.update(pit={"id": pit["id"], "keep_alive": keep_alive})
```

Comment:
```suggestion
        search._extra.update(pit={"id": pit_id, "keep_alive": keep_alive})
```
since you have already declared id of the PIT